.vscode/settings.json

```diff
@@ -5,6 +5,7 @@
     "source.dcm.fixAll": "explicit"
   },
   "dart.flutterSdkPath": ".fvm/versions/3.19.0",
+  "dart.lineLength": 80,
```

Comment:
We understand it's a personal option. However, we are inclined to go with the previous option.

Reply:
pub.dev requires such line length. Why not add this rule to have correct formatting without a separate command?